cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js

```diff
@@ -12,42 +11,51 @@ import Checkout from '../../../support/fragments/checkout/checkout';
 import AppPaths from '../../../support/fragments/app-paths';
 import LoanDetails from '../../../support/fragments/users/userDefaultObjects/loanDetails';
 import LoansPage from '../../../support/fragments/loans/loansPage';
-import RequestPolicy from '../../../support/fragments/circulation/request-policy';
-import OverdueFinePolicy from '../../../support/fragments/circulation/overdue-fine-policy';
-import LostItemFeePolicy from '../../../support/fragments/circulation/lost-item-fee-policy';
-import NoticePolicy from '../../../support/fragments/circulation/notice-policy';
 import RenewConfirmationModal from '../../../support/fragments/users/loans/renewConfirmationModal';
 import OverrideAndRenewModal from '../../../support/fragments/users/loans/overrideAndRenewModal';
 import Loans from '../../../support/fragments/users/userDefaultObjects/loans';
 import CheckInActions from '../../../support/fragments/check-in-actions/checkInActions';
 import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
 import Users from '../../../support/fragments/users/users';
 import DevTeams from '../../../support/dictionary/devTeams';
+import LoanPolicyActions from '../../../support/fragments/circulation/loan-policy';
+import CirculationRules from '../../../support/fragments/circulation/circulation-rules';
+import Location from '../../../support/fragments/settings/tenant/locations/newLocation';
 
 describe('ui-users-loans: renewal failure because loan has reached maximum renewals', () => {
   const loanTypeName = `autotest_loan_type${getRandomPostfix()}`;
   const newFirstItemData = getNewItem();
   const newSecondItemData = getNewItem();
+  const servicePoint = ServicePoints.getDefaultServicePointWithPickUpLocation('autotest', uuid());
+  const servicePointId = servicePoint.id;
```

Comment:
Why do you need this variable? 
You can actually use servicePoint.id

Reply:
Done

Reply:
Done